### ISSUE TITLE
Error: _TypeError (Null check operator used on a null value)

### ISSUE THREAD

**Issue by OttomanDeveloper** [raysummee/expire_it#8]
![Screenshot 2024-01-26 230224](https://github.com/raysummee/expire_it/assets/71749423/53d5bdd5-6a29-4375-99f5-e62922cd835a)

Getting this weird error:

```
void main() async {
  WidgetsFlutterBinding.ensureInitialized();
  ExpireIt.controller = UserDefineExpireController(
    expiryDate: DateTime(2024),
  );
  runApp(const App());
}
```

**Comment by raysummee** [comment]
Hi @OttomanDeveloper

Sorry for late reply. I noticed many confusion when implementing this library. I'll soon make changes so that the confusion won't be there in future.

Regarding your problem. Please make sure you wrapping the parent widget with ExpireIt Widget.

```dart
ExpireIt(
    expireBuilder: (context) {
        return const Center(
            child: Text("Expired"),
        );
    },
    builder: (context) {
        return const Center(
            child: Text("Home screen"),
        );
    },
    loadingBuilder: (context) {
        return const Center(
            child: CircularProgressIndicator(),
        );
    },
),
```

**Comment by raysummee** [comment]
Issue has been fixed by #9 PR. 

The latest ref is at master, tagged v1.0.0
